flexget/plugins/list/ombi_list.py

```diff
@@ -231,17 +230,36 @@ def items(self):
                                             self._items.append(entry)
         return self._items
 
+    def add_movie(self, entry):
+        log.verbose('Adding entry %s', entry.get('title'))
+        auth_header = self.ombi_auth()
+
+        parsedurl = urlparse(self.config.get('base_url'))
+        url = '%s://%s:%s%s/api/v1/Request/movie' % (parsedurl.scheme, parsedurl.netloc, self.config.get('port'), parsedurl.path)
```

Comment:
I notice you've hardcoded this in a lot of places and I apologize for not bringing it up sooner.

1. Use global/module variables for the endpoints similar to this https://github.com/Flexget/Flexget/blob/develop/flexget/plugins/list/sonarr_list.py#L16-L20. You can then use a function like this https://github.com/Flexget/Flexget/blob/develop/flexget/plugins/internal/api_trakt.py#L200-L211 to generate a proper URL.
2. You use urlparse a lot but always on the same url. You should only do it once and re-use the values. You don't have to use it though. We have a json schema validator `"format": "url"` that you can use and then it's a simple matter of `url.rstrip('/')` to remove any trailing slashes.